### PR TITLE
infra: bump golang to 1.19 on openshift-ci image

### PR DESCRIPTION
**Pull request by SchSeba**
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: openshift-ci/Dockerfile.tools [hunk 1]
```diff
@@ -28,10 +28,9 @@ RUN mkdir ~/bin && \
     eval "$(gimme $GOVERSION)" && \
     cat $GIMME_ENV >> $HOME/.bashrc && \
     # get required golang tools and OC client
-    go get github.com/onsi/ginkgo/ginkgo@v1.16.5 && \
-    go get github.com/onsi/gomega/...@v1.17.0 && \
-    go get -u golang.org/x/lint/golint && \
-    go install github.com/lack/mcmaker@v0.0.5 && \
+    go install github.com/onsi/ginkgo/v2/ginkgo@latest && \
```
Comment (fedepaol) [inline openshift-ci/Dockerfile.tools:31]: is this wanted? Here we are also bumping ginkgo to v2

**Comment by fedepaol** [comment]
nit on the commit subject: on openshifT

File: openshift-ci/Dockerfile.tools (hunk 1 above)
Comment (fedepaol) [reply openshift-ci/Dockerfile.tools:31]: that's fine, but maybe it belongs to a separate commit, because the commit says "bump go"

Comment (SchSeba) [reply openshift-ci/Dockerfile.tools:31]: agree! :)

**Pull request merged**